src/components/flame-graph/Canvas.js

```diff
@@ -204,7 +214,7 @@ class FlameGraphCanvas extends React.PureComponent<Props> {
         if (w2 > textMeasurement.minWidth) {
           const fittedText = textMeasurement.getFittedText(funcName, w2);
           if (fittedText) {
-            ctx.fillStyle = isHovered ? 'HighlightText' : foreground;
+            ctx.fillStyle = highlightBox ? 'HighlightText' : foreground;
```

Comment:
I think it might be better to have two colors, one for selected (using `Highlight`) and then another for the hover, as the colors are now ambiguous in their meanings. Perhaps we can darken or lighten the normal color? I adapted some code from another library that could work here.

```js
function changeColorBrightness (hexString, factor) {
  if (hexString.length !== 7 || hexString[0] !== '#') {
    throw new Error('Expect a hex string of the form "#000000"');
  }

  // Go from a string of the form "#000000" to a number of the form 0x000000.
  const hex = parseInt(hexString.substr(1), 16);

  // Take the hex value, and get R G B values ranged 0-255.
  let r = (0xff0000 & hex) >> 16;
  let g = (0x00ff00 & hex) >> 8;
  let b = (0x0000ff & hex);

  // Apply the darkening or lightening.
  r = Math.max(0, Math.min(255, Math.floor(r * factor)));
  g = Math.max(0, Math.min(255, Math.floor(g * factor)));
  b = Math.max(0, Math.min(255, Math.floor(b * factor)));

  // Turn this back into a value like 0x000000, but include an extra bit so it takes the
  // form 0x1000000 and can keep the leading zeros.
  const modifiedHex = 0xf000000 | (r << 16) | (g << 8) | b;
  
  return '#' + modifiedHex.toString(16).substr(1);
}

describe('changeColorBrightness', function() {
  it('can increase the brightness', function() {
    expect(changeColorBrightness('#334455', 2.0)).toBe('#6688aa')
  });
  it('cannot increase the brightness over white', function() {
    expect(changeColorBrightness('#334455', 100.0)).toBe('#ffffff')
  });
  it('can decrease the brightness', function() {
    expect(changeColorBrightness('#334455', 0.5)).toBe('#19222a')
  });
  it('cannot go below black', function() {
    expect(changeColorBrightness('#334455', -0.5)).toBe('#000000')
  });
});
```

Reply:
Thanks! I tried with your function, but when I darkened the colors, they lost saturation I think (not sure of the terminology). They looked a bit grey. Increasing the brightness make the colors a bit too bright.

So what I did instead was to to pick the next color in the range provided by the photon design system. Hope you like it :)